### PR TITLE
feat: add volta completion spec

### PR DESCRIPTION
**Pull request by Monkeyank**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature
**What is the current behavior? (You can also link to an open issue here)**
No completion spec for `volta`
**What is the new behavior (if this is a feature change)?**
Add a completion spec for `volta`. 
**Additional info:**
- [Volta Docs](https://docs.volta.sh/reference/)
- Fulfills #240

### CODE REVIEW

File: dev/volta.ts [hunk 1]
```diff
@@ -0,0 +1,188 @@
+const defaultOptions = [
```
Comment (QuiiBz) [inline dev/volta.ts:1]: I would love to see this typed as a `Fig.Option[]`

File: dev/volta.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        },
+      ],
+    },
+    {
+      name: "which",
+      description: "Locates the actual binary that will be called by Volta",
+      args: {
+        variadic: true,
+        name: "binary",
+      },
```
Comment (QuiiBz) [inline dev/volta.ts:111]: Doesn't this can have a template of `filepaths`?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for your first contribution! I left some changes before we can get this merged in.

File: dev/volta.ts [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+      options: [...defaultOptions],
+    },
+    {
+      name: "install",
+      description: "Installs a tool in your toolchain",
+      args: {
+        variadic: true,
+        name: "tool@version",
+      },
```
[the remark this answers is not in the document]
Comment (Monkeyank) [reply dev/volta.ts:36]: Good catch! Didn't notice it at first glance

**Review by cstrnt — APPROVED** [review]
Thank you very much :blush

**Review by QuiiBz — APPROVED** [review]
Let's get this merged in!

**Pull request merged**